common/views/components/BasePage/EventPage.js

```diff
@@ -129,22 +144,154 @@ const EventPage = ({ event }: Props) => {
     >
       <Fragment>
         {event.schedule && event.schedule.length > 0 &&
-          <Fragment>
+          <div className='body-text'>
             <h2 className={`${font({s: 'WB6', l: 'WB5'})} ${spacing({s: 4}, {padding: ['bottom']})} border-color-smoke border-bottom-width-2`}>Events</h2>
             <ul className='plain-list no-marin no-padding'>
               {event.schedule && event.schedule.map((scheduledEvent) => {
                 return (<EventScheduleItem key={scheduledEvent.id} event={Object.assign({}, scheduledEvent, {description: scheduledEvent.promo && scheduledEvent.promo.caption})}
                   hasOwnPage={Boolean(scheduledEvent.description)} />);
               })}
             </ul>
-          </Fragment>
+          </div>
+        }
+
+        {event.series.map((series) => {
+          return (
+            <div className='body-text' key={series.id}>
+              <h2>Part of <a href='/event-series/{series.id}'>{series.title}</a></h2>
+              <div dangerouslySetInnerHTML={{__html: series.description}} />
```

Comment:
We can avoid using this pattern by keeping the parsed type as [`HTMLString`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/types.js#L57).

This is useful as then the renderer can decide how it want to render it - we can also use the [`PrismicHTMLBlock`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/views/components/PrismicHtmlBlock/PrismicHtmlBlock.js) for consistency.